src/main/java/com/woongeya/zoing/domain/application/presetation/ApplicationController.java

```diff
@@ -23,4 +25,7 @@ public void createApplication(@RequestBody ApplicationCreateRequest request, @Pa
     public void cancelApplication(@PathVariable Long id) {
         applicationCancelService.execute(id);
     }
+
+    @PutMapping("/reject/{id}")
```

Comment:
```suggestion
    @PutMapping("/{id}/reject")
```

이런 방식으로 하는 게 1번 application을 reject 한다는 의미가 더 잘 드러나지 않을까요?

`/application/1/reject` vs `/application/reject/1`